R/plumber.R

```diff
@@ -288,6 +290,8 @@ plumber <- R6Class(
       debug = interactive(),
       swaggerCallback = getOption('plumber.swagger.url', NULL)
     ) {
+      if (isTRUE(private$disable)) {return()}
```

Comment:
@blairj09  

If trying to run a plumber router while plumbing a file, would you expect it to 
* Skip over the run method
* Present a message / warning
* Throw an error
?

My thought is to throw. This will strongly discourage this _bad_ coding pattern.

Reply:
Agreed

Reply:
Let's throw then

Reply:
Can we also change the flag name to something a little longer? `$disable_run` ?

Thank you, @meztez !